resources/user-agents/crawlers/general-crawlers/general-crawlers.json

```diff
@@ -1986,6 +1986,10 @@
         {
           "match": "Mixmax-LinkPreview *",
           "browser": "mixmax link preview"
+        },
+        {
+          "match": "Mozilla/5.0 (*Linux*x86_64*) AppleWebKit/* (KHTML, like Gecko) splash Version/* Safari/*",
+          "browser": "splash"
```

Comment:
This might be better if it were in the "rippers" group

Reply:
moved